src/components/FeedCard/FeedCard.stories.js

```diff
@@ -0,0 +1,10 @@
+import FeedCard from './FeedCard';
+
+export default {
+  title: 'FeedCard',
```

Comment:
```suggestion
  title: 'components/FeedCard',
```

구현해주신 컴포넌트의 스토리 파일의 루트 경로가 `FeedCard`로 잡혀있네요.
모두 `components/SomeComponent...`처럼 해당 내용 수정해주셔야 할 것 같습니다!

Reply:
[969a330](https://github.com/prgrms-web-devcourse/Team_KPPL_STUTI_FE/pull/27/commits/969a33018e0a628533ca57d36da085ed843ba9e8)
주의 하겠습니다. :)